platform_plugin_forum_email_notifier/email.py

```diff
@@ -0,0 +1,27 @@
+"""Email module for platform_plugin_forum_email_notifier"""
+from django.contrib.sites.models import Site
+from edx_ace import ace
+from edx_ace.message import MessageType
+from openedx.core.lib.celery.task_utils import emulate_http_request
+
+
+class ForumEmailNotification(MessageType):
+    """
+    Message type for forum email notification.
+    """
+
+
+forum_email_message_type = ForumEmailNotification()
```

Comment:
Why initialize this here?

Reply:
ACE messages contain a global context that's used to fulfill basic information, this is done at startup, but we are not using it. We can move it if needed